README.md

```diff
@@ -184,60 +202,57 @@ My name is Vlad. Буде цікаво!
 Glad to join this group. I wish you all the fulfillment of your desires and dreams.
 This is my repository(https://github.com/Ros1989/kottans-frontend). Subscribe to my GitHub https://github.com/Ros1989
 
-
 #Hello!
 My name is Kyryl.
 I want to complete all tasks and go to Stage 1.
 
 [Link](https://github.com/kirill8210/kottans-frontend) to my repo on GitHub.
 
 #My cat is watching me))
+
 <p>
 ![Introduction Sequence](./assets/images/chester.jpg)
 </p>
 
-
-﻿# Hey there!
-
+# Hey there!
 
 ## Hi there, I'm Stella!
+
 ![Сміливий котик](https://cdn.abo.media/upload/article/res/770-430/o_1fvlg5oeb1gvq1v4rruvaq4gpccj.jpg)
-_*I wish everyone inspiration and endurance.*
-____
+\__I wish everyone inspiration and endurance._
```

Comment:
It seems like you need to configure your IDE not to adjust .md files like this.
It would be nice if you could only change only the lines you want to add.

Reply:
how can i customize the IDE

Reply:
You can google how to customize your personal IDE in any way you like. But for this particular task, it will be just easier to undo your commits, add changes again, but if you are using VSCode (as you should), you can use the "Save without formatting" command. You see, VSCode auto-formats files on save and we have a mess in this file here on purpose, so students can learn not to touch anything that they don't need to.

You can google about undoing changes in git with a `git revert` command.

Right now your PR has 157 additions and 120 deletions. If the task is done correctly you should have almost no deletions and only your greetings in additions. To see all changes that you're trying to make with this PR, open the "[Files changed](https://github.com/kottans/mock-repo/pull/1061/files)" tab.